src/main/java/com/uid2/shared/secure/azurecc/MaaTokenPayload.java

```diff
@@ -0,0 +1,26 @@
+package com.uid2.shared.secure.azurecc;
+
+import lombok.Builder;
+import lombok.Value;
+
+@Value
+@Builder(toBuilder = true)
+public class MaaTokenPayload {
+    public static final String SEV_SNP_VM_TYPE = "sevsnpvm";
+    public static final String AZURE_COMPLIANT_UVM = "azure-compliant-uvm";
+
+    private String attestationType;
+    private String complianceStatus;
+    private boolean vmDebuggable;
+    private String ccePolicy;
```

Comment:
Actually it's "hostData", should we rename it as "hostData", otherwise it's a bit confusing because in the deployment template "ccePolicy" means the actual policy instead of policy digest

Reply:
Or "ccePolicyDigest"?

Reply:
Will change it to ccePolicyDigest.